src/server.cc

```diff
@@ -332,12 +345,12 @@ int Server::handleUpnpEvent(Upnp_EventType eventtype, const void* event)
     return ret;
 }
 
-std::string Server::getIP() const
+std::string Server::getIP()
```

Comment:
Did you mean to remove the const?

Reply:
good point. there is a place in the code where getIP and getPort is called. indead of passing through the server via shared_ptr, i simply made getIP and getPort static. but now "const" is no longer allowed...